source/includes/steps-enable-authentication-in-shardcluster-nodowntime-transition-mongos.yaml

```diff
@@ -0,0 +1,79 @@
+stepnum: 1
+level: 4
+title: "Add ``transitionToAuth: true`` to the ``mongos`` configuration file"
+ref: modify-config-files
+pre: |
+  
+  Copy the existing {{program}} configuration file, giving it a distinct
+  name such as ``<filename>-secure``.
+  
+  You will use this new configuration file to transition the {{program}} to
+  enforce authentication in the sharded cluster. Retain the original
+  configuration file for backup purposes.
+
+  Add the following settings to the new configuration file.
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 30 70
+
+     * - Setting
+       - Value
+
+     * - :setting:`security.transitionToAuth`
+       - ``true``
+
+     * - :setting:`security.keyFile`
+       - Path to the keyfile.
+
+         If using a different internal authentication mechanism, specify
+         settings appropriate for the mechanism.
+
+  The new configuration file should contain all of the configuration settings
+  previously used by the {{program}}, as well as the new security
+  settings.
+
+  .. code-block:: yaml
+
+     security:
+        transitionToAuth: true
+        keyFile: <path-to-keyfile>
+replacement:
+  program: :program:`mongos`
+---
+stepnum: 2
+level: 4
+ref: restart-mongos
+title: "Restart the ``mongos`` with the new configuration file."
+pre: |
+
+  Restart one :program:`mongos` at a time to ensure that clients can connect
+  to the sharded cluster. If your cluster has only one :program:`mongos`, this
```

Comment:
This step makes it seem like I would restart each of the mongos at step 2 instead of doing steps 1 & 2 for one mongos before repeating on the next mongos.  Which contradicts the section intro paragraph.  Pick one or the other.  If you want people to update all the conf first and then restart one by one, or update the conf, restart, then update the conf for the other and restart.